packages/react-conformance/src/defaultTests.tsx

```diff
@@ -65,12 +67,95 @@ export const defaultTests: TestObject = {
   },
 
   /** Constructor/component name matches filename */
-  'name-matches-filename': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
+  'name-matches-filename': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
     it(`Component/constructor name matches filename`, () => {
       const { componentPath, displayName } = testInfo;
       const fileName = path.basename(componentPath, path.extname(componentPath));
 
       expect(displayName).toEqual(fileName);
     });
   },
+
+  /** Ensures component is exported at top level allowing import { Component } from 'packageName' */
+  'exported-top-level': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    if (!testInfo.isInternal) {
+      it(`is exported at top-level`, () => {
+        const { displayName, componentPath, Component } = testInfo;
+        const rootPath = componentPath.replace(/src.*/, '');
+        const indexFile = require(path.join(rootPath, 'src', 'index'));
+
+        expect(indexFile[displayName]).toBe(Component);
+      });
+    }
+  },
+
+  /** Ensures component has top level file in package/src/componentName */
+  'has-top-level-file': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    if (!testInfo.isInternal) {
+      it(`has corresponding top-level file 'package/src/Component'`, () => {
+        const { displayName, componentPath, Component } = testInfo;
+        const rootPath = componentPath.replace(/src.*/, '');
+        const topLevelFile = require(path.join(rootPath, 'src', displayName));
+
+        expect(topLevelFile.default || topLevelFile[displayName]).toBe(Component);
+      });
+    }
+  },
+
+  /** If the component is a subcomponent, ensure its parent has the subcomponent as static property */
+  'is-static-property-of-parent': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`is a static property of its parent`, () => {
+      const { componentPath, displayName, useDefaultExport } = testInfo;
+      const componentFolder = componentPath.replace(path.basename(componentPath) + path.extname(componentPath), '');
+      const dirName = path.basename(componentFolder).replace(path.extname(componentFolder), '');
+      const isParent = displayName === dirName;
+
+      if (!isParent) {
+        const subComponentName = displayName.replace(dirName, '');
+        const subComponentFile = require(path.join(componentFolder, subComponentName));
+        const SubComponent = useDefaultExport ? subComponentFile.default : subComponentFile[subComponentName];
+
+        expect(ReactIs.isValidElementType(SubComponent)).toBe(true);
+      }
+    });
+  },
+
+  /** Ensures aria attributes are kebab cased */
+  'kebab-aria-attributes': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`Aria attributes are kebab-cased`, () => {
+      const props = Object.keys(componentInfo.props);
+
+      for (const prop of Object.keys(props)) {
```

Comment:
I'm pretty sure this isn't working as expected--`props` is an array, so `Object.keys(props)` will return the indices. Instead you can just do `for (const prop of props)`.

In general, for each test, it's a good idea to temporarily make a component violate the criteria to ensure that the test actually catches violations.